fetch-members.js

```diff
@@ -0,0 +1,52 @@
+const { GraphQLClient } = require("graphql-request");
+const { writeFileSync } = require("fs");
+
+async function fetchMembers(organisation) {
+  const token = process.env.GH_TOKEN;
+
+  if (!token) {
+    console.error("'GH_TOKEN' not set. Could not fetch nteract members.");
+    return [];
+  }
+
+  const client = new GraphQLClient("https://api.github.com/graphql", {
+    headers: {
+      Authorization: `Bearer ${token}`
+    }
+  });
+
+  const query = `{
+    organization(login: ${organisation}) {
+      members(first: 100) {
+        totalCount
+          nodes {
+            name
+            login
+            websiteUrl
+            avatarUrl
+            url
+          }
+      }
+    }
+  }`;
+
+  try {
+    const data = await client.request(query);
+    if (data.organization.members.totalCount > 100) {
+      console.error(
+        "100+ members in the organization. That's too much for one GraphQL call."
+      );
```

Comment:
I suppose this means we can go ahead and page through several queries, slightly defeating the point of GraphQL for "only fetching what you need, once". 😆

Reply:
We're at ~40 public members. But in the future we may need to do several queries.